CHANGES.md

```diff
@@ -35,6 +35,18 @@ To be released.
     `HashDigest<SHA256>?`.  [[#3446], [#3447]]
  -  (Libplanet.Explorer) Added `TxResult.OutputState` of type
     `HashDigest<SHA256>?`.  [[#3446], [#3447]]
+ -  (Libplanet.Explorer) Added `offsetBlockHash` of type
+    `HashDigest<SHA256>?` argument for `StateQuery.states` field.
+    [[#3448], [#3449]]
+ -  (Libplanet.Explorer) Added `offsetBlockHash` of type
+    `HashDigest<SHA256>?` argument for `StateQuery.balance` field.
+    [[#3448], [#3449]]
+ -  (Libplanet.Explorer) Added `offsetBlockHash` of type
+    `HashDigest<SHA256>?` argument for `StateQuery.totalSupply` field.
+    [[#3448], [#3449]]
+ -  (Libplanet.Explorer) Added `offsetBlockHash` of type
+    `HashDigest<SHA256>?` argument for `StateQuery.validators` field.
+    [[#3448], [#3449]]
```

Comment:
You mean `offsetStateRootHash`? 😶

Reply:
Shade 😓

Reply:
https://github.com/planetarium/libplanet/pull/3449/commits/3005e4e27858fdd3c1dee65aac636f8f80d3e669 amended.